libs/libcommon/cmake/find_jemalloc.cmake

```diff
@@ -1,4 +1,4 @@
-if (OS_LINUX AND NOT SANITIZE)
+if (OS_LINUX AND NOT SANITIZE AND NOT ARCH_ARM)
```

Comment:
Better to add proper configuration files for jemalloc on ARM
(otherwise we'll get very suboptimal code)